src/Pages/Main/Component/MainSection1.scss

```
@@ -0,0 +1,38 @@
+section {
+  margin: 0;
+  padding: 70px 0;
+  width: 100%;
+  height: auto;
+}
+
+section:nth-child(1) {
```

```suggestion
section {
  &:nth-child(1)
```
이렇게 하면 section 안쪽으로 넣어줄 수 있습니다